src/pybind/mgr/restful/module.py

```diff
@@ -198,6 +198,7 @@ class Module(MgrModule):
         {'name': 'server_addr'},
         {'name': 'server_port'},
         {'name': 'key_file'},
+        {'name': 'disable_auth'},
```

Comment:
would be better to specify the type and the default value of this setting, like
```suggestion
        {'name': 'disable_auth', 'type': 'bool', 'default', False},
```

but personally, i think a positive setting would be easier to read, something like
```suggestion
        {'name': 'enable_auth', 'type': 'bool', 'default', True},
```